modules/ROOT/partials/individually-licensed-components/import-from-word-and-export-to-word/import-from-word-and-export-to-word-autorization-on-premises.adoc

```diff
@@ -17,7 +17,7 @@ If the `SECRET_KEY` is not setup during the installation, then {pluginname} On-P
 * it is not issued in the future (e.i. the `iat` timestamp cannot be newer than the current time),
 * it has not expired yet.
 
-Tokens for the {pluginname} On-Premises do not require any additional claims such as the environment ID (which is specific for Collaboration Server On-Premises), the token can be created with an empty payload.
+Tokens for the {pluginname} On-Premises do not require any additional claims such as the aud (which is specific to cloud-based document converters), so the token can be created with an empty payload.
```

Comment:
```suggestion
Tokens for the {pluginname} On-Premises do not require any additional claims such as the `aud` (which is specific to cloud-based document converters), so the token can be created with an empty payload.
```

This should be marked as a code tag because it's a code property name